tests/languages/nsis/constant_feature.test

```diff
@@ -0,0 +1,13 @@
+$(myLicenseData)
+${LANG_ENGLISH}

```

Comment:
One of your changes in the component was to include `.` and `:` in the allowed characters. Could you add examples using those?